src/bin/psql/model.h

```diff
@@ -0,0 +1,32 @@
+#ifndef MODEL_H_4D955674
```

Comment:
Its a uuid of sorts - I have been bitten by this before, where two headers have the same short name (and same ifdef guard). So by default, I set up my editor to add a unique guard.

```
#include "a/foo.h"
#include "b/foo.h"
```